src/library_glfw.js

```diff
@@ -753,6 +753,7 @@ var LibraryGLFW = {
 
       event.preventDefault();
 
+#if '$FS' in addedLibraryItems
```

Comment:
Interesting, I didn't know about this "preprocessor syntax"

Reply:
Yeah, we made it possible to run arbitrary JS in the preprocessor, I think this was a few months ago.